pandas-stubs/io/parsers/__init__.pyi

```diff
@@ -0,0 +1,7 @@
+from pandas.io.parsers.readers import (

```

Comment:
I know the change here is to remove `io/parsers.pyi` and replace with a directory `parsers`, but for some reason the removal of `io/parsers.pyi` is not showing up as a "changed file" .  Could be a github bug, but can you check that the `git rm io/parsers.pyi` is in your commit?